src/shared_definitions.hpp

```diff
@@ -248,20 +249,20 @@ using IndexSpec = seqan3::default_sdsl_index_type;
 //                                 sdsl::plain_byte_alphabet>;
 
 /* epr, in experimental branch */
-// template <size_t alph_size>
-// using IndexSpec = sdsl::csa_wt<sdsl::wt_epr<alph_size + 2>, // +1 for sentinels, +1 for collection
-//                                 16,
-//                                 1'0000'000,
-//                                 sdsl::sa_order_sa_sampling<>,
-//                                 sdsl::isa_sampling<>,
-//                                 sdsl::plain_byte_alphabet>;
+template <size_t alph_size>
+using IndexSpec = sdsl::csa_wt<sdsl::wt_epr<alph_size + 2>, // +1 for sentinels, +1 for collection
+                                2,
+                                1'0000'000,
+                                sdsl::sa_order_sa_sampling<>,
+                                sdsl::isa_sampling<>,
+                                sdsl::plain_byte_alphabet>;
 
 // ==========================================================================
 //  Misc. aliases
 // ==========================================================================
 
 template <typename TString>
-using TCDStringSet = std::vector<TString>; //TODO seqan3::concatenated_sequences
+using TCDStringSet = seqan3::concatenated_sequences<TString>; //TODO seqan3::concatenated_sequences
```

Comment:
I have changed this now and needed to change a few other things for that. It might give us a bit of performance with larger data (but I couldn't verify in my laptop).